openssl3/tools/checksyms.bash

```diff
@@ -28,6 +28,7 @@ ALLOWED_SYMS="$ALLOWED_SYMS _init _lib_version _PROCEDURE_LINKAGE_TABLE_"
 # Because we let OpenSSL 3 do the library renaming, it includes this in the
 # symbol list. OpenSSL 3 will be adding to this as it adds new symbols.
 ALLOWED_SYMS="$ALLOWED_SYMS OPENSSL_SMARTOS_3.0.0 OPENSSL_SMARTOS_3.0.3"
+ALLOWED_SYMS="$ALLOWED_SYMS OPENSSL_SMARTOS_3.0.8"
```

Comment:
We don't need `OPENSSL_SMARTOS_3.0.0` anymore?

Reply:
We actually do:
```
smartos-build(~/OS-8442)[0]% nm proto/lib/libcrypto-smartos.so | grep OPENSSL_SMARTOS
[20122] |         0|         0|OBJT |GLOB |3    |ABS    |OPENSSL_SMARTOS_3.0.0
[20126] |         0|         0|OBJT |GLOB |3    |ABS    |OPENSSL_SMARTOS_3.0.3
[20134] |         0|         0|OBJT |GLOB |3    |ABS    |OPENSSL_SMARTOS_3.0.8
smartos-build(~/OS-8442)[0]% 
```

Reply:
I don’t understand why it was removed from this line then.

Reply:
The line wasn't removed.  This is an ADDITIONAL LINE (see how it still includes the prior value of ALLOWED_SYMS inside it?).

Reply:
This is a bash script so there might be a cleaner way to add items on multiple lines, but given how close we are to branching, maybe next time around?